backend/typescript/services/implementations/__tests__/campService.test.ts

```diff
@@ -1,403 +1,403 @@
-import db from "../../../testUtils/testDb";
```

Comment:
The tests were failing since I changed the create function. We don't really test so I propose we just remove these since they are so flaky